llvm/include/llvm/IR/Cheri.h

```diff
@@ -55,6 +56,68 @@ inline bool isCheriPointer(Type *Ty, const DataLayout *DL) {
 }
 
 namespace cheri {
+
+/// Represents the LLVM-level attribute that is used to signal
+/// cross-compartment imports.
+class CapabilityImportAttr {
```

Comment:
Since it seems like this is only ever used to fetch the two attribute name and the final string, maybe this could just be a function that returns an `std::pair`?

Reply:
Actually, I see it's used for more than that in other places, so I take this comment back.